software/playground.py

```diff
@@ -11,4 +11,39 @@
 
 bb.measure()
 print(bb.cellVolts)
-print(bb.cellTemps)
\ No newline at end of file
+print(bb.cellTemps)
+
+#blame Lev below this line
+from datetime import datetime
+import csv
+
+#function takes in timestamp and voltage, writes values to csv file, "a" appends whereas "w" clears file first
```

Comment:
Docstrings should be  under function def and block comments

Reply:
If this means that the comments for each function should be under the function while non-function-specific comments should be above, then we should be good now.